Stylesheets/components/badge.css

```diff
@@ -0,0 +1,73 @@
+@import url(/Stylesheets/main.css);
+
+.bedge {
+  margin: 2rem;
+}
+.bg {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  height: 2rem;
+  width: 2rem;
+  background-color: red;
+  border-radius: 50%;
+  position: relative;
```

Comment:
The badge should have position absolute and its parent should have position relative

Reply:
yes done as u said

Reply:
You have to give position:relative to parent which is img here and position absolute to badge.
You will need to change your HTML structure for it.
Your badges are outside the parent image container.
go through this
[https://stackify.dev/386106-css-badge-over-image-with-bootstrap](url)